test/reporter.spec.js

```diff
@@ -0,0 +1,66 @@
+import { expect } from "expect";
+import SVGLint from "../src/svglint.js";
+
+async function lint(source, rules) {
+    const linting = await SVGLint.lintSource(source, { rules });
+    return new Promise(resolve => {
+        linting.on("done", () => {
+            resolve(linting.results);
+        });
+    });
+}
+
+describe("Reporter", function() {
+    it("should report result without node", async function() {
+        const results = await lint(
+            "<svg><title></title></svg>",
+            { elm: { "svg > foobar": true } }
+        );
+        expect(results.elm.messages).toMatchObject([{
+            message: "Expected 'svg > foobar', none found",
+            reason: "Expected 'svg > foobar', none found",
+            type: "error"
+        }]);
+    });
+
+    it("should report results with node", async function() {
+        const results = await lint(
+            `<svg>
+               <g>
+                 <path />
+                 <path />
+               </g>
+               <g></g>
+             </svg>`,
+            { elm: { "g > path": false } }
+        );
+        expect(results.elm.messages).toMatchObject([{
+            message: "Element disallowed\n  At node \u{1B}[1m<path>\u{1B}[22m (2:17)",
```

Comment:
I think we can just use [`chalk`](https://www.npmjs.com/package/chalk) here, like in [the source](https://github.com/birjj/svglint/blob/834abc486612349a9d4f30e8d7990506ddc0d96c/src/lib/reporter.js#L44), to ensure ANSI styling is applied consistently between the implementation and test assertions. That should fix the remaining [CI errors](https://github.com/birjj/svglint/actions/runs/6676161047/job/18161511290).